docs/platform-support.md

```diff
@@ -3,3 +3,4 @@
 FakeItEasy officially supports the following platforms:
 
 * .NET Framework 4.0 onwards
+* .NET Standard 1.6

```

Comment:
Initially .NET Standard 2.0 was going to be breaking, but after a public backlash they decided to make it backwards compatible. But perhaps that will change with 3.0?

The .NET Framework hasn't had a breaking release in its 15 year history so I think that's less likely to happen in the future than with .NET Standard, but who knows, maybe it will happen for the first time with 5.0?

Ultimately, what the above demonstrates, is that we can't second-guess what's going to happen with future major version bumps in the platforms. That is entirely under the control of the publishers of those platforms.  I think we should state here, as we do in the NuGet package metadata, what specific versions we support. The backwards compatibility of future versions of those platforms is documented elsewhere. That means we should remove "onwards".

Phew, that's a long justification for removing the word "onwards". 😳